From af13da7551ab7d7782758e50915e7267e09d1e34 Mon Sep 17 00:00:00 2001
From: Shengzhou Liu <Shengzhou.Liu@freescale.com>
Date: Mon, 29 Dec 2014 17:32:31 +0800
Subject: [PATCH 52/57] t102x/diu: move display alias to t1024si-post.dtsi

As DIU/display feature exists on t1024 instead of t1023,
so move display alias to t1024si-post.dtsi.

Signed-off-by: Shengzhou Liu <Shengzhou.Liu@freescale.com>
Change-Id: I831f211f8f85142622e63706204df2265f80ed0d
Reviewed-on: http://git.am.freescale.net:8181/32939
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Xiaobo Xie <X.Xie@freescale.com>
Reviewed-by: Honghua Yin <Hong-Hua.Yin@freescale.com>
[Original patch taken from QorIQ-SDK-V1.8-20150619-yocto]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/powerpc/boot/dts/fsl/t1024si-post.dtsi |    7 +++++++
 arch/powerpc/boot/dts/fsl/t102xsi-pre.dtsi  |    2 --
 2 files changed, 7 insertions(+), 2 deletions(-)

diff --git a/arch/powerpc/boot/dts/fsl/t1024si-post.dtsi b/arch/powerpc/boot/dts/fsl/t1024si-post.dtsi
index 8eae871..13948bb 100644
--- a/arch/powerpc/boot/dts/fsl/t1024si-post.dtsi
+++ b/arch/powerpc/boot/dts/fsl/t1024si-post.dtsi
@@ -34,6 +34,13 @@
 
 /include/ "t1023si-post.dtsi"
 
+/ {
+	aliases {
+		vga = &display;
+		display = &display;
+	};
+};
+
 &soc {
 /include/ "qoriq-tdm1.0.dtsi"
 
diff --git a/arch/powerpc/boot/dts/fsl/t102xsi-pre.dtsi b/arch/powerpc/boot/dts/fsl/t102xsi-pre.dtsi
index e2b27b9..4b4756a 100644
--- a/arch/powerpc/boot/dts/fsl/t102xsi-pre.dtsi
+++ b/arch/powerpc/boot/dts/fsl/t102xsi-pre.dtsi
@@ -67,8 +67,6 @@
 		ethernet1 = &fm1mac2;
 		ethernet2 = &fm1mac3;
 		ethernet3 = &fm1mac4;
-		vga = &display;
-		display = &display;
 	};
 
 	cpus {
-- 
1.7.5.4

